### DISCUSSION TITLE
Error: Vectors must have the same length

### DISCUSSION THREAD

**Discussion by tapangarg** [brianpetro/obsidian-smart-connections#738] [Q&A]
![Screenshot 2024-08-25 054446](https://github.com/user-attachments/assets/49c8c150-9369-401f-b434-4d894c57bdbf)

Newbie user, I'm getting this error - Vectors must have the same length when I try to view the smart connections of any note. Please help.

**Comment by tapangarg** [comment]
I reinstalled everything after removing the old smart connections directories. Working fine now.

**Reply by christianngarcia** [reply]
Thanks for following up. 😊
